packages/react/src/components/Toggle/Toggle.tsx

```diff
@@ -191,7 +191,7 @@ export function Toggle({
         role="switch"
         type="button"
         aria-checked={checked}
-        aria-labelledby={ariaLabelledby ?? labelId}
+        aria-labelledby={ariaLabelledby ?? (labelText ? labelId : '')}
```

Comment:
Would it make sense for it to be `false` or `undefined` instead of an empty string? That way the attribute won't be placed on the element at all